doc/content/getting-started/migrating-from-v2/mqtt-api-comparison.md

```diff
@@ -77,4 +77,4 @@ weight: 70
 
 [TTI V2 MQTT API Documentation](https://www.thethingsnetwork.org/docs/applications/mqtt/api.html)
 
-[TTES V3 MQTT API Documentation](https://enterprise.thethingsstack.io/v3.8.1/integrations/mqtt/)
+[TTS V3 MQTT API Documentation](https://enterprise.thethingsstack.io/v3.8.1/integrations/mqtt/)
```

Comment:
We have dropped version number from documentation links right? Also, should this not be a ref instead?